qiskit/transpiler/passes/calibration/rzx_builder.py

```diff
@@ -206,7 +206,7 @@ def get_calibration(self, node_op: CircuitInst, qubits: List) -> Union[Schedule,
                     "RZX schedule is not generated for this qubit pair.",
                     UserWarning,
                 )
-            raise CalibrationNotAvailable
+            raise CalibrationNotAvailableError
```

Comment:
I _think_ these are only ever internal signals, but if there's any chance of them propagating to users, it's an API break to change its name.

Reply:
All of these reverted. They can be revisited perhaps.